polls/models.py

```diff
@@ -44,26 +47,39 @@ class Poll(models.Model):
     # The date of publication.
     publication_date = models.DateField(
         verbose_name='date de publication',
-        blank=True,
         null=True,
+        default=None,
     )
 
     # A comment about the poll from the administrator.
     admin_comment = models.TextField(
         verbose_name='commentaire administrateur',
         blank=True,
+        default='',
     )
 
+    @cached_property
     def has_been_published(self):
         """
-        Return True if the poll is has been published in the past, that is publication_date_time <= now(),
-        and its state is ACCEPTED.
+        Return True if the poll is has been published in the past, that is publication_date <= now(), and its state is
+        ACCEPTED.
         """
         td = date.today()
         if self.publication_date is None or self.state != 'ACCEPTED':
             return False
         return self.publication_date <= td
```

Comment:
No need to declare a tmp variable that will be read once, not modified and short to write